src/Factory/CarFactory.php

```diff
@@ -0,0 +1,31 @@
+<?php
+namespace App\Factory;
+
+use App\Entity\Car;
+use App\Entity\User;
+
+class CarFactory
```

Comment:
you don't have the abstract factory/factory method at all. The point of pattern is not implemented at all. Abstraction is nowhere. Read about pattern again